### PR TITLE
n2-3 Implement swagger error responses for News

### PR DESCRIPTION
**Pull request by dmkofgl**

### CODE REVIEW

File: src/main/java/dl/news/portal/web/config/SwaggerConfig.java [hunk 1]
```diff
@@ -1,36 +1,89 @@
...
+                .additionalModels(new TypeResolver().resolve(LinkResponse.class))
+                .additionalModels(new TypeResolver().resolve(ErrorResponse.class))
+                .directModelSubstitute(LinkResponse.class, Link.class);
+    }
+
+    private static <T> List<T> concatenatedList(List<T>... collections) {
+        return Arrays.stream(collections).flatMap(Collection::stream).collect(Collectors.toList());
+    }
+
+    private List<ResponseMessage> message422Validate() {
```
Comment (avinahradau) [inline src/main/java/dl/news/portal/web/config/SwaggerConfig.java]: Replace `422` with verbal error designation

File: src/main/java/dl/news/portal/web/config/SwaggerConfig.java [hunk 2]
```diff
@@ -1,36 +1,89 @@
...
+        return Arrays.asList(new ResponseMessageBuilder()
+                .code(HttpStatus.UNPROCESSABLE_ENTITY.value())
+                .message("Validation error")
+                .responseModel(new ModelRef("BindExceptionResponse"))
+                .build());
+    }
+
+    private List<ResponseMessage> message404() {
```
Comment (avinahradau) [inline src/main/java/dl/news/portal/web/config/SwaggerConfig.java]: Same here

**Review by avinahradau — CHANGES_REQUESTED** [review]

File: src/main/java/dl/news/portal/web/config/SwaggerConfig.java (hunk 1 above)
Comment (dmkofgl) [reply src/main/java/dl/news/portal/web/config/SwaggerConfig.java]: Done

File: src/main/java/dl/news/portal/web/config/SwaggerConfig.java (hunk 2 above)
Comment (dmkofgl) [reply src/main/java/dl/news/portal/web/config/SwaggerConfig.java]: Done

**Review by avinahradau — APPROVED** [review]

**Pull request merged**